src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/contentgrid/viewmodel/ViewContentViewModel.kt

```diff
@@ -0,0 +1,95 @@
+package org.wycliffeassociates.otter.jvm.app.ui.contentgrid.viewmodel
+
+import com.github.thomasnield.rxkotlinfx.changes
+import com.github.thomasnield.rxkotlinfx.observeOnFx
+import com.github.thomasnield.rxkotlinfx.toObservable
+import io.reactivex.Observable
+import javafx.beans.property.SimpleBooleanProperty
+import javafx.beans.property.SimpleObjectProperty
+import javafx.collections.FXCollections
+import javafx.collections.ObservableList
+import org.wycliffeassociates.otter.common.data.model.Collection
+import org.wycliffeassociates.otter.common.data.model.Content
+import org.wycliffeassociates.otter.common.domain.content.AccessTakes
+import org.wycliffeassociates.otter.jvm.app.ui.inject.Injector
+import tornadofx.*
+
+class ViewContentViewModel: ViewModel() {
```

Comment:
let's let the viewmodel name match the view:
ContentGridViewModel

especially since "ViewContent" becomes confusing because in the same name you have two different meanings of the word View, which becomes confusing, but also the word View just begins to be noise and unmeaningful

Reply:
you're right. I missed that change when i changed the other names